File: Aprendendo-C++/Exercícios/soma_impares.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int n,m, aux = 0, somarImp= 0;
    
cout << "Digte dois numeros\n";
cin >> n; //maior
cin >> m; 

if(m > n){
    aux = m; 
    m = n;
    n =aux;
}

for( int i = m+1 ; n > i; i++){
    if (i%2 != 0 ){
        somarImp += i;
    }
    cout << m+1;
    cout << i;
}

cout << "Soma dos impares: " << somarImp << endl;

    return 0;
}

File: Aprendendo-Python/Exercícos/soma_impares.py
print("Digite dois numeros: ")
x = int(input())
y = int(input())

aux: int = 0
somarImp: int = 0;

if(y < x):
    aux = y
    y = x
    x = aux


for i in range((x + 1),y):
    if (i % 2 != 0):
        somarImp += i

print("Soma dos pares: ",somarImp)